### ISSUE TITLE
2 different accessibility standards for the same test during the same AET run

### ISSUE THREAD

**Issue by RdkLwg** [Cognifide/aet#499]
I would like to be able to run two accessibility health checks for the same test (e.g. WCAG2.0 and Section508) for the same run.
It will be great if you could also define a name for the Accessibility tab, as it is possible in case of visual comparison e.g.:
- "Accessibility WCAG2.0 AA",
- "Accessibility Section508".

Note that no filters applied for one standard should be displayed for the other.
